cmd/gpu_plugin/README.md

```diff
@@ -5,17 +5,9 @@ Table of Contents
 * [Introduction](#introduction)
     * [Configuration options](#configuration-options)
 * [Installation](#installation)
-    * [Deploy with pre-built container image](#deploy-with-pre-built-container-image)
-    * [Getting the source code](#getting-the-source-code)
-    * [Deploying as a DaemonSet](#deploying-as-a-daemonset)
-        * [Build the plugin image](#build-the-plugin-image)
-        * [Deploy plugin DaemonSet](#deploy-plugin-daemonset)
-        * [Fractional resources](#fractional-resources)
-    * [Deploy by hand](#deploy-by-hand)
-        * [Build the plugin](#build-the-plugin)
-        * [Run the plugin as administrator](#run-the-plugin-as-administrator)
-    * [Verify plugin registration](#verify-plugin-registration)
-    * [Testing the plugin](#testing-the-plugin)
+    * [Pre-built Images](#pre-built-images)
+    * [Verify Plugin Registration](#verify-plugin-registration)
```

Comment:
Link to "Fractional resources" section 8that still remains) is missing.

Reply:
fixed in the next rev